hedera-node/hapi-fees/src/test/resources/simple-fee-schedule.json

```diff
@@ -58,7 +58,8 @@
             {  "name": "SIGNATURES", "includedCount": 1 },
             {  "name": "KEYS",       "includedCount": 1 },
             {  "name": "ACCOUNTS",       "includedCount": 1 },
-            {  "name": "STANDARD_FUNGIBLE_TOKENS", "includedCount": 1}
+            {  "name": "STANDARD_FUNGIBLE_TOKENS", "includedCount": 1},
+            {  "name": "STANDARD_NON_FUNGIBLE_TOKENS", "includedCount": 1}
```

Comment:
These should be included only for `CryptoTransfer+SubType` transaction right? How are we deciding subtypes